### ISSUE TITLE
[BUG] all my files and folder names have been changed into lower case

### ISSUE THREAD

**Issue by coursegence** [vrtmrz/obsidian-livesync#487]
hundreds of my files and folders have been renamed all into lower case.

I now have to manually rename them all :(

**Comment by vrtmrz** [comment]
Thank you for opening the issue! And I am so sorry for that.

However, unfortunately, I could not reproduce it yet.
Would you share the report, which we can put out on the setting dialogue?

**Comment by lzism** [comment]
I also encountered the same problem, case insensitivity causes synchronization to other devices to become lowercase

**Comment by lzism** [comment]
> I also encountered the same problem, case insensitivity causes synchronization to other devices to become lowercase
And it caused some confusion, now I have two folders, inbox and Inbox, and there are different files in them, maybe it is my use, but this function should be said more carefully

**Comment by lzism** [comment]
> I also encountered the same problem, case insensitivity causes synchronization to other devices to become lowercase

And it caused some confusion, now I have two folders, inbox and Inbox, and there are different files in them, maybe it is my use, but this function should be said more carefully

**Comment by alx-xlx** [comment]
I agree this is a very drastic change, it will possibly break all other extension settings, across the vault. Almost all plugins depend on the case-sensitive files and folder names. example templates, paths etc. 

I don't think case-sensitivity should be treated as a feature that has to be enabled.

**Comment by vrtmrz** [comment]
Oh my goodness, it seems to be a common problem. So sorry for all of you! 
By the design, a case should be held even if checking files case-insensitively. All of you are on Linux? 
I am now on the mobile phone, I will check the behaviour and address it as soon as I get back to the terminal.  Apologies for your inconvenience and troubles!

**Comment by vrtmrz** [comment]
Now I backed to the desktop and checked the behaviour with several patterns of vaults. 
I finally found the actual reason that Retrieving files on the start-up scanning could not handle the filename case fine. This will be fixed soon! Sorry for your inconvinience!
If you have the backup, or are not synchronised with the remote with corrupted files, your vault will be restored by fetching the remote into the empty vault with the next release.

**Comment by vrtmrz** [comment]
I just released v0.23.22; which fixes this problem!

Would you mind if I ask you to check problems were fixed correctly, please? Fetching again into the empty vault might be effective.

**Comment by coursegence** [comment]
thank you so much, it is now fixed. The update has fixed the issue luckily it didn't affected other device as it was offline. 
You really saved me !!! 🥇

**Comment by coursegence** [comment]
my main vault was on windows and I had fetched it to ios

**Comment by coursegence** [comment]
also just a heads up, the Database cleaning has some issues
```
9/9/2024, 10:59:39 AM->Database active-size: 33 MB, external-size:61 MB, file-size: 38 MB
9/9/2024, 10:59:39 AM->Collecting unreferenced chunks on mydb
9/9/2024, 10:59:40 AM->No unreferenced chunks! mydb
9/9/2024, 10:59:40 AM->Processed database active-size: 33 MB, external-size:61 MB, file-size: 38 MB
9/9/2024, 10:59:40 AM->Reduced sizes: active-size: NaN undefined, external-size:-, file-size: NaN undefined
```
it gives `NaN undefined` error
its unable to clean my remote chunks, its unreasonably high for some reason. it used to be around 5000chunks, but now its 12000chunks. Not that it matters much but just a heads up :)

**Comment by hermanmak** [comment]
I seem to have this same issue on the latest version of obsidian livesync 0.24.11 on mobile and desktop.

The files are duplicated except the capitals. Is my vault malformed now?

**Comment by eitchtee** [comment]
Had the same problem on the 0.25.2 when doing the first fetch on an Android device from Windows.

Forcefully rebuilding the database on Windows with the original files and re-fetching fixed the issue. My database and vault were created many versions ago so it was probably outdated or something.
